README.md

```diff
@@ -84,7 +84,7 @@ Our code development features a mix of upstream OpenStack work and custom ESI co
 
 ## Contact and Contribution
 
-You can contact the ESI development team on the Freenode `#moc` IRC channel.
+You can contact the ESI development team on the OFTC `#moc` IRC channel.
```

Comment:
As long as we're updating this document, we should probably make `OFTC` a link to https://www.oftc.net/

Reply:
Good point - done!